### ISSUE TITLE
Title of upload video is always "Untitled"

### ISSUE THREAD

**Issue by Ant-Adis** [vimeo/VimeoUpload#241]
#### Issue Summary
I Use a really basic approach for this and try 3 times and every time tittle is "Untitled" even If I change VideoSetting in code, not sure what to do here. I can not run your examples because Xcode gives me some errors after build. The description is also missing, only the privacy works as I can see.
P.S.
Im using this Pod:
pod 'VimeoUpload', :git => 'git@github.com:vimeo/VimeoUpload.git', :branch => 'develop'

#### Reproduction Steps
```
    override func viewDidLoad() {
        super.viewDidLoad()
        guard let path = Bundle.main.path(forResource: "joy", ofType:"mp4") else {
            debugPrint("video.m4v not found")
            return
        }
        let urlll = URL(fileURLWithPath: path)
        let title = "New Title!"
        let description = "A really cool video"
        let privacy = "public"
        let videoSettings = VideoSettings(title: title, description: description, privacy: privacy, users: nil, password: nil)

        let descriptor = OldUploadDescriptor(url: urlll, videoSettings: videoSettings) //OldUploadDescriptor(url: fileUrl, videoSettings: self.videoSettings)
        descriptor.identifier = "AnotherOne"
        descriptor.addObserver(self, forKeyPath: type(of: self).StateKeyPath, options: .new, context: &self.stateKVOContext)
        descriptor.addObserver(self, forKeyPath: type(of: self).ProgressKeyPath, options: .new, context: &self.progressKVOContext)


        OldVimeoUploader.sharedInstance?.uploadVideo(descriptor: descriptor)
    }
```

#### Expected Behavior
Title to be the one I enter

#### Actual Behavior
Title is always: "Untitled"

**Comment by rpassis** [comment]
Hi there,

For what it's worth the old upload manager is no longer supported but I've tried this myself with the example app and have been able to upload a video with a custom title.

One thing to note on the code above is that your privacy `public` value is invalid - you are probably looking for `anybody` instead. Setting it to public will cause your request to be rejected so that may be where things are going wrong and possibly causing some confusion.

Could you please retry the above with the correct privacy value and let me know how that goes?

If you still can't make it work please add some details on what your URLRequest object looks like, including the serialized (JSON) settings object.

Thanks!
Rog

**Comment by Ant-Adis** [comment]
Hmm, strange, first I tried with default like in your project example: `privacy: "nobody"` but that did not work, now I search a little more and found that you have constants for privacies and I try this:
privacy: `VIMPrivacy_Private`
and that works :S

I tried to create your `NewVimeoUploader.sharedInstance?.uploadVideo`
but I could not figure out where You get this:` let video = self.video! `property. 
I know it is from `ExportSessionExportCreateVideoOperation` but that is when user choose from album we must create a copy of asset and then upload it, but how when user record himself, I already have video (asset) don't need to do `ExportSessionExportCreateVideoOperation`
I need to upload the video when a user record himself and when user choose video from the album.
Thanks
